docs/src/privileged.md

```diff
@@ -0,0 +1,24 @@
+# Privilege Escalation
+
+Some actions may be run either privileged or unprivileged. For those unfamiliar, this would be by utilizing sudo or
+running something as admin. A common action this is utilized with is [command.run](./command). Perhaps you, as the user,
+wish to run a command that comtrya does not directly supply an action for, but that command required some form of
+privilege escalation, some actions allow that to be specified. Here is an example utilizing `command.run`.
+
+```
+- action: command.run
+    command: whoami
```

Comment:
Identation here seems a little wrong. Command and args should be -2. Same below

Reply:
Will fix that. Didn't catch that my text editor decided to format it for me.